examples/interactive.jl

```diff
@@ -713,4 +834,6 @@ function record_example(title = "Orbit Diagram")
     end
     AbstractPlotting.use_display[] = last
 end
-# record_example("Volume slider")
+
+
+# record_example("Lighting")
```

Comment:
@ffreyer, when you return RecordEvents in an example, you can call this function with the title of the example to make a recording!

Reply:
Hmm, I think my mistake was that I didn't run `database = MakieGallery.load_database();`. Without it `record_example` doesn't even exist.

Anyway, I think it would be good to keep the light source in view when messing with theta and phi. That'd make it easier to see what's happening.